notebooks/ch11/gradient_descent_define.md

```diff
@@ -379,41 +354,41 @@ np.mean([12.1, 12.8, 14.9, 16.3, 17.2])
 
 
 
-### Minimizing the Huber cost
+### Minimizing the Huber loss
 
-Now, we can apply gradient descent to minimize the Huber cost on our dataset of tip percentages.
+Now, we can apply gradient descent to minimize the Huber loss on our dataset of tip percentages.
 
-The Huber cost is:
+The Huber loss is:
 
 $$
-L_\delta(\theta, y) = \frac{1}{n} \sum_{i=1}^n \begin{cases}
-    \frac{1}{2}(y_i - \theta)^2 &  | y_i - \theta | \le \delta \\
-     \delta (|y_i - \theta| - \frac{1}{2} \delta ) & \text{otherwise}
+L_\delta(\hat{\theta}, y) = \frac{1}{n} \sum_{i=1}^n \begin{cases}
+    \frac{1}{2}(y_i - \hat{\theta})^2 &  | y_i - \hat{\theta} | \le \delta \\
+     \delta (|y_i - \hat{\theta}| - \frac{1}{2} \delta ) & \text{otherwise}
```

Comment:
probably fine, but this equation is not parsing in nbviewer. just fyi in case we find out it doesnt show up on gitbook either